plugins/wasmedge_ffmpeg/avutil/pixfmt.cpp

```diff
@@ -0,0 +1,199 @@
+#include "pixfmt.h"
+extern "C" {
+#include "libavutil/pixdesc.h"
+}
+
+namespace WasmEdge {
+namespace Host {
+namespace WasmEdgeFFmpeg {
+namespace AVUtil {
+
+// Expect<int32_t> AVPixFmtDescGet::body(const Runtime::CallingFrame
```

Comment:
Why does this section comment out? If we don't need it, remove it. Otherwise, it would be best if you kept it unimplemented or something meaningful.